### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sophyting**
Just a first pass to add more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
-    access_id {lm_access_id}
-    access_key {lm_access_key}
-    flush_interval 1s
-    debug false
+@type lm
+company_name <your_company_name>
+resource_mapping {"<event_key>": "<lm_property>"}
+access_id <your_lm_access_id>
+access_key <your_lm_access_key>
+
```
Comment (ahsan13jan) [inline README.md:22]: Is this space intentional?  And adding tab before the arguments would be better.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,56 @@
...
-## Getting started
-- Add `out_lm.rb` to plugins directory
+## Prerequisites
 
-- Add to `fluent.conf`
+Add `out_lm.rb` to your Fluentd plugins directory.
+
+## Configure the output plugin
+
+Create a custom `fluent.conf` or edit the existing one to specify which logs should be forwarded to LM.
```
Comment (mchlstckl) [inline README.md:11]: Do we ever use `LM` in public communication?

File: README.md [hunk 3]
```diff
@@ -1,31 +1,56 @@
...
+flush_interval 1s
+debug false
 </match>
 ```
 
-To send data by
+### Request example
 
+Sending:
 `curl -X POST -d 'json={"message":"hello Logic Monitor from fluentd", "event_key":"lm_property_value"}' http://localhost:8888/lm.test`
```
Comment (mchlstckl) [inline README.md:31]: Logic Monitor -> LogicMonitor

File: README.md (hunk 1 above)
Comment (sophyting) [reply README.md]: it was, but I guess there's no reason for it. Removed the space and added tabs.

**Pull request merged**